addons/binding/org.openhab.binding.systeminfo/.classpath

```diff
@@ -4,9 +4,9 @@
     path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8" />
   <classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins" />
   <classpathentry kind="src" path="src/main/java" />
-  <classpathentry exported="true" kind="lib" path="lib/jna-platform-4.4.0.jar" />
-  <classpathentry exported="true" kind="lib" path="lib/jna-4.4.0.jar" />
-  <classpathentry exported="true" kind="lib" path="lib/oshi-core-3.4.3.jar" />
-  <classpathentry exported="true" kind="lib" path="lib/threetenbp-1.3.5.jar" />
+  <classpathentry exported="true" kind="lib" path="lib/jna-platform-4.5.0.jar" />
+  <classpathentry exported="true" kind="lib" path="lib/jna-4.5.0.jar" />
```

Comment:
According to the [changelog](https://github.com/oshi/oshi/blob/master/CHANGELOG.md#350-41518) OSHI 3.5.0 depends on JNA 4.5.1.

Reply:
Thank you very much for review. According to openHab log, at least on Raspberry, JNA 4.5.1 will not work properly: 
![grafik](https://user-images.githubusercontent.com/19878123/39667706-60f4b166-50bc-11e8-9555-d250d966fdaf.png)

Downgrade to JNA 4.5.0 worked. There is a similar error, i think, but not sure about, for oshi 3.4.4, is reported here: https://github.com/oshi/oshi/issues/463